flow2ml/Data_Augumentation.py

```diff
@@ -109,4 +109,18 @@ def applyShear(self,classPath):
             # saving the image by
             plt.imsave(classPath+"/ShearedImages/Sheared"+image, cv2.cvtColor(Sheared, cv2.COLOR_RGB2BGR))
           except Exception as e:
-            print(f"Shearing operation failed due to {e}")
\ No newline at end of file
+            print(f"Shearing operation failed due to {e}")
+
+  
+  def HistogramEqualisation(image_file):
```

Comment:
Take self as one of parameter(just to follow code standards)
And also add documentation in proper format for arguments that you are using